Nikita_Konkov_Application/CSharpAdvanced/CSharpAdvanced/Books/BookInfo.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpOOP2
+{
+    static class BookInfo
+    {
+        public static float CostPerPage (this Book bookIt) 
+        {
+            return bookIt.Cost / bookIt.Pages;
+        }
+        
+        public static void InfoBookIt(this Book bookIt)
+        {
+             Console.WriteLine($"Here we have {bookIt.Tittle} that was written by {bookIt.Author} in {bookIt.Date}. In book {bookIt.Pages}. And the cost of the book just {bookIt.Cost} BYN"); 
+        }
+
+        public static double BooItCostWithDiscaunter(this Book bookIt, double discount = 0.17)
```

Comment:
I love the fact that you followed code standards and create extension class only for 1 data type! well done!